libs/core/src/components/pds-popover/stories/pds-popover.stories.js

```diff
@@ -0,0 +1,56 @@
+import { html } from 'lit';
+import { extractArgTypes } from '@pxtrn/storybook-addon-docs-stencil';
+import { withActions } from '@storybook/addon-actions/decorator';
+
+export default {
+  argTypes: extractArgTypes('pds-popover'),
+  args: {
+    hasArrow: false
+  },
+  component: 'pds-popover',
+  decorators: [withActions],
+  parameters: {
+    actions: {
+      handles: ['pdsPopoverShow', 'pdsPopoverHide'],
+    },
+  },
+  title: 'components/Popover'
+}
+
+const BaseTemplate = (args) => html`
+  <pds-popover has-arrow=${args.hasArrow} placement=${args.placement}>
```

Comment:
In each story, the opened toggle doesn't work as expected when interacting with the controls.